### DISCUSSION TITLE
Gcsfuse need to sync 1 way, from local to GCS or from GCS to local only

### DISCUSSION THREAD

**Discussion by InoriYuzuriha0** [GoogleCloudPlatform/gcsfuse#1015] [Q&A]
Hi All,

I had some question about gcsfuse, in my case I want to create sync folder from local(local, instance VM, or another persistent
 storage), but I want to sync 1 way(from local to GCS) or from GCS to local, not 2 way sync. Any suggestion with my case, configuration I need to apply?
 
 Thank you,
 
 Yogi

**Comment by vadlakondaswetha** [comment]
If you are going to sync from GCS to local and not going to make any writes, then you can use read only mount by passing -o r
But for syncing from local to GCS or GCS to local with writes, use a regular mount (readwrite mount).

Either way you need to use implicit-dirs flag if your use-case falls as mentioned here: https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/docs/semantics.md#implicit-directories

More details about what flags to use while mounting is mentioned here: https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/docs/semantics.md

**Comment by raj-prince** [comment]
Closing this as didn't get response from the user for a long time.
